### ISSUE TITLE
When using a multi-polygon to restrict the tile area for mapcache_seed, only the first part is used

### ISSUE THREAD

**Issue by mapcentia** [mapserver/mapcache#202]
When using something like this, where the layer `dk_buffer_g` is a multi part polygon from PostGIS, seeding will stop after tiles within the first part of the polygon is done.

`mapcache_seed -c baselayers.xml -t geodk.bright -g g20 -z 18,20 -d PG:"host=some_host user=gc2 dbname=baselayers password=xx" -l "dk_buffer_g"`

I would expect the whole multi-polygon would be used?

**Comment by jbo-ads** [comment]
Yes the whole multi-polygon should be used. I tried with a simple multi-polygon (four non-overlapping rectangles) saved in a GeoJSON file and couldn't reproduce this issue.
